It looks like the failures on aarch64 are not new:
https://hydra.nixos.org/build/202124533
https://hydra.nixos.org/build/202185637

```
configure: error: platform "xen" is not supported for target CPU "arm64"
```

--- a/nixos/tests/installer.nix
+++ b/nixos/tests/installer.nix
@@ -291,7 +291,7 @@ let
           # builds stuff in the VM, needs more juice
           virtualisation.diskSize = 8 * 1024;
           virtualisation.cores = 8;
-          virtualisation.memorySize = 1536;
+          virtualisation.memorySize = 2047;
 
           boot.initrd.systemd.enable = systemdStage1;
 

